index.html

```diff
@@ -1,343 +1,358 @@
+              <path fill-rule="evenodd"
+                d="M5.293 7.293a1 1 0 011.414 0L10 10.586l3.293-3.293a1 1 0 111.414 1.414l-4 4a1 1 0 01-1.414 0l-4-4a1 1 0 010-1.414z"
+                clip-rule="evenodd" />
+            </svg>
+          </div>
+          <ul class="language">
+            <li class="lang">
+              <img alt="js" src="/assets/js.png">
+              <p>JavaScript</p>
+            </li>
+            <li class="lang">
+              <img alt="html" src="/assets/html.png">
+              <p>HTML</p>
+            </li>
+            <li class="lang">
+              <img alt="css" src="/assets/css.png">
+              <p>CSS</p>
+            </li>
+          </ul>
+        </div>
+        <div class="language-section">
+          <div class="dropdown-menu border-b">
+            <p class="lang-text">Frameworks</p>
+            <svg xmlns="http://www.w3.org/2000/svg" class="icon" viewBox="0 0 20 20" fill="currentColor">
+              <path fill-rule="evenodd"
+                d="M7.293 14.707a1 1 0 010-1.414L10.586 10 7.293 6.707a1 1 0 011.414-1.414l4 4a1 1 0 010 1.414l-4 4a1 1 0 01-1.414 0z"
+                clip-rule="evenodd" />
+            </svg>
+          </div>
+        </div>
+        <div class="language-section">
+          <div class="dropdown-menu border-b">
+            <p class="lang-text">Skills</p>
+            <svg xmlns="http://www.w3.org/2000/svg" class="icon" viewBox="0 0 20 20" fill="currentColor">
+              <path fill-rule="evenodd"
+                d="M7.293 14.707a1 1 0 010-1.414L10.586 10 7.293 6.707a1 1 0 011.414-1.414l4 4a1 1 0 010 1.414l-4 4a1 1 0 01-1.414 0z"
+                clip-rule="evenodd" />
+            </svg>
+          </div>
+        </div>
+      </div>
+    </section>
+    <!--CONTACT FORM SECTION  -->
+    <footer class="contact-section-bg">
+      <div class="contact-section">
+        <h2 class="title">Contact me</h2>
+        <p class="details">If you have an application you are interested in developing, a feature that you need
+          built or a project that needs coding. I’d love to help with it </p>
+        <form action="https://formspree.io/f/mjvjzbal" method="POST" class="form">
+          <ul>
+            <li class="form-field">
+              <input placeholder="Name" class="input" type="text" minlength="1" maxlength="30" name="name" required>
+            </li>
+            <li class="form-field">
+              <input placeholder="Email" class="input" type="email" name="email" required>
+            </li>
+            <li class="form-field">
+              <textarea placeholder="Write your message here" class="text-area" name="message" required rows="10"
+                maxlength="500"></textarea>
+            </li>
+            <li>
+              <button class="button" type="submit">Get In Touch</button>
+            </li>
+          </ul>
+        </form>
+      </div>
```

Comment:
- Please try to stick as much as possible to the Figma design, which includes: 
   - [ ] Using the same font style ( color, family, size, weight, ...etc) that is used in the Figma design.
   - [ ] Adjusting the spacing around the elements so that it matches the Figma design.

| Your design | Figma design |
| --- | --- |
| ![image](https://user-images.githubusercontent.com/62290545/159911080-13eead1c-60a1-41f4-a305-5d789788c837.png) | ![image](https://user-images.githubusercontent.com/62290545/159911000-2e0f5b58-e41c-415f-b693-ad29ab897220.png) |